app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt

```diff
@@ -361,38 +365,58 @@ class PaymentAdapter(
                 data: PaymentModel,
                 marketProvider: MarketProvider,
                 fragmentManager: FragmentManager,
-                tracker: PaymentTracker
+                tracker: PaymentTracker,
             ) = Unit
         }
 
-        class PayoutDetailsParagraph(parent: ViewGroup) :
-            PaymentAdapter.ViewHolder(parent.inflate(R.layout.payout_details_paragraph)) {
-            private val binding by viewBinding(PayoutDetailsParagraphBinding::bind)
+        class PayoutConnectionStatus(parent: ViewGroup) :
+            ViewHolder(parent.inflate(R.layout.payout_connection_status)) {
+            private val binding by viewBinding(PayoutConnectionStatusBinding::bind)
             override fun bind(
                 data: PaymentModel,
                 marketProvider: MarketProvider,
                 fragmentManager: FragmentManager,
-                tracker: PaymentTracker
+                tracker: PaymentTracker,
             ) = with(binding) {
-                if (data !is PaymentModel.PayoutDetailsParagraph) {
+                if (data !is PaymentModel.PayoutConnectionStatus) {
                     return invalid(data)
                 }
 
-                root.setText(
-                    when (data) {
-                        PaymentModel.PayoutDetailsParagraph.Add -> R.string.payment_screen_pay_out_footer_not_connected
+                when (data.status) {
+                    PayoutMethodStatus.ACTIVE -> {
+                        root.setText(R.string.payment_screen_pay_connected_label)
+                        root.putCompoundDrawablesRelativeWithIntrinsicBounds(
+                            start = R.drawable.ic_checkmark_in_circle
+                        )
```

Comment:
```suggestion
                        root.putCompoundDrawablesRelativeWithIntrinsicBounds(start = R.drawable.ic_checkmark_in_circle)
```

As such? 🙂

Reply:
@hugokallstrom

Reply:
Sorry I meant between `root.setText(R.string.payment_screen_pay_connected_label)` and 
```
root.putCompoundDrawablesRelativeWithIntrinsicBounds(
                            start = R.drawable.ic_checkmark_in_circle
                        )
```